apps/meteor/tests/e2e/page-objects/login.ts

```diff
@@ -23,6 +23,10 @@ export class LoginPage {
 		await expect(this.page.getByRole('main')).toBeVisible();
 	}
 
+	async waitForLogout() {
+		await expect(this.loginButton).toBeVisible();
+	}
```

Comment:
What is the difference between this method and the `waitForIt` method?

Reply:
None, I just don't agree we should deprecate the `waitForIt`, a nice way to check if the login was successful is by checking some piece on the login, which IMO should live on the `LoginPage` class, so for me doesn't make sense to deprecate it

Reply:
I agree with you. Previously I thought that at some near future the login flow would be improved enough to the point we wouldn't rely on awaiting a condition to proceed. Maybe we should remove the deprecation tag but keep the comment attached to it.